OneSignalSDK/onesignal/notifications/src/main/java/com/onesignal/notifications/internal/permissions/impl/NotificationPermissionController.kt

```diff
@@ -64,7 +70,22 @@ internal class NotificationPermissionController(
             )!!
 
     init {
+        _enabled = notificationsEnabled()
         _requestPermission.registerAsCallback(PERMISSION_TYPE, this)
+        coroutineScope.launch {// Should this be Android version restricted?
```

Comment:
Seems like the AndroidX library' `areNotificationsEnabled() takes care of all Android versions:
https://cs.android.com/androidx/platform/frameworks/support/+/androidx-main:core/core/src/main/java/androidx/core/app/NotificationManagerCompat.java;l=364-389?q=notificationmanagercompat